index.Rmd

```diff
@@ -318,11 +319,11 @@ plot2=leaflet() %>%
 
 ## Methods - Raster Analysis
 
-NLCD Raster data were used to determine the percent of deciduous and mixed (deciduous/coniferous) forest that covered each county. The code to calculate this data was shown earlier, but a detailed explanation (which ignores the loop) is shown below.
+NLCD Raster data were used to determine the percent of deciduous and mixed (deciduous/coniferous) forest that covered each county. The code to calculate this data was shown earlier,  but a detailed explanation (which ignores the loop) is shown below.
 
 Data for each county is downloaded via this code:
 ``` {r explanation1,eval=F,echo=T}
-        NLCD=get_nlcd(template = st_as_sf(newcounties[i,]),
+        NLCD=get_nlcd(template = st_as_sf(newcounties[i,]), 
```

Comment:
So when I tried to build the website on my end, I actually got an error here. Here is what it said: 
Error in get_nlcd(template = st_as_sf(newcounties[i, ]), year = 2016,  : 
  unused argument (landmass = "L48")
Calls: <Anonymous> ... handle -> withCallingHandlers -> withVisible -> eval -> eval
In addition: Warning message:
All elements of `...` must be named.
Did you want `data = c(Year, Pathogen, Life_Stage, Total_tested, Tot_pos, percpos)`? 
Execution halted

Exited with status 1.

As a result, I couldn't build the website myself, so this project sadly isn't reproducible yet.

Reply:
This code isn't meant to be run, it's only meant to show up to explain how the loop works. The only get_nlcd() line that is supposed to be run in the loop on line 268. I'm surprised it isn't running though. Did you try to both "build website" and "knit"?

Reply:
> This code isn't meant to be run, it's only meant to show up to explain how the loop works. The only get_nlcd() line that is supposed to be run in the loop on line 268. I'm surprised it isn't running though. Did you try to both "build website" and "knit"?

When I knit, it gives me back the same error.

Reply:
I think I fixed it. I believe it was a crs problem with that shapefile. I think the updated version of my Index.Rmd should work now (hopefully)

Reply:
I just redownloaded you zip file and I still got this error:
Error in get_nlcd(template = st_as_sf(newcounties[i, ]), year = 2016,  : 
  unused argument (landmass = "L48")
Calls: <Anonymous> ... handle -> withCallingHandlers -> withVisible -> eval -> eval
In addition: Warning message:
All elements of `...` must be named.
Did you want `data = c(Year, Pathogen, Life_Stage, Total_tested, Tot_pos, percpos)`? 
Execution halted

Exited with status 1.